### PR TITLE
Check model_info_ instead of is_finished_

### PR DESCRIPTION
**Pull request by fujunwei**
Fixes:
https://github.com/intel/webml-polyfill/issues/77
https://github.com/intel/webml-polyfill/issues/89
https://github.com/intel/webml-polyfill/issues/174

which is regression by
https://github.com/otcshare/chromium-src/pull/20

### CODE REVIEW

**Comment by fujunwei** [comment]
@sunlin-link How about using this PR to fix the regression?

**Comment by fujunwei** [comment]
@huningxin PTAL, thanks.

File: third_party/blink/renderer/modules/ml/model.cpp [hunk 1]
```diff
@@ -18,44 +18,40 @@
 namespace blink {
 
 namespace {
-bool InvalidState(bool is_finished,
-                  const String& message,
-                  ExceptionState& exception_state) {
-  String error_message = is_finished ? "Model is finished." : "";
```
Comment (huningxin) [inline third_party/blink/renderer/modules/ml/model.cpp]: I just think "Model is finished" is a useful error message to let user know. Could we keep that?

Comment (fujunwei) [reply third_party/blink/renderer/modules/ml/model.cpp]: Make sense, thanks, done.

**Comment by huningxin** [comment]
I would suggest to keep `is_finished_` to indicate the `finish` method is called by JavaScript. We may need different error messages.

**Comment by fujunwei** [comment]
Thanks, done.

**Comment by huningxin** [comment]
Looks good. Thanks!

**Pull request merged**